Threshold level 1e-2 starting
Traceback (most recent call last):
  File "bucket_classifier.py", line 4, in <module>
    import pandas as pd
ImportError: No module named pandas
Threshold level 1e-2 complete

Threshold level 1e-1 starting
Traceback (most recent call last):
  File "bucket_classifier.py", line 4, in <module>
    import pandas as pd
ImportError: No module named pandas
Threshold level 1e-1 complete

Threshold level 2 starting
Traceback (most recent call last):
  File "bucket_classifier.py", line 4, in <module>
    import pandas as pd
ImportError: No module named pandas
Threshold level 2 complete

Threshold level 5 starting
Traceback (most recent call last):
  File "bucket_classifier.py", line 4, in <module>
    import pandas as pd
ImportError: No module named pandas
Threshold level 5 complete

Threshold level 7 starting
Traceback (most recent call last):
  File "bucket_classifier.py", line 4, in <module>
    import pandas as pd
ImportError: No module named pandas
Threshold level 7 complete

Threshold level 10 starting
Traceback (most recent call last):
  File "bucket_classifier.py", line 4, in <module>
    import pandas as pd
ImportError: No module named pandas
Threshold level 10 complete

Threshold level 12 starting
Traceback (most recent call last):
  File "bucket_classifier.py", line 4, in <module>
    import pandas as pd
ImportError: No module named pandas
Threshold level 12 complete

Threshold level 15 starting
Traceback (most recent call last):
  File "bucket_classifier.py", line 4, in <module>
    import pandas as pd
ImportError: No module named pandas
Threshold level 15 complete

Threshold level 18 starting
Traceback (most recent call last):
  File "bucket_classifier.py", line 4, in <module>
    import pandas as pd
ImportError: No module named pandas
Threshold level 18 complete

Threshold level 20 starting
Traceback (most recent call last):
  File "bucket_classifier.py", line 4, in <module>
    import pandas as pd
ImportError: No module named pandas
Threshold level 20 complete

